### ISSUE TITLE
Js error due to null selection when viewing source via setKeystroke hotkey (patch)

### ISSUE THREAD

**Issue by themanyone** [ckeditor/ckeditor4#3998]
## Type of report

Bug, patch widget plulgin.js

## Provide detailed reproduction steps (if any)

1. Set up a hotkey for "view source" in config.js

```
editor1.setKeystroke([
        [ CKEDITOR.CTRL + 13 /*Enter*/, 'source' ],
]);
```
2. Press the hotkey

### Expected result
No error

### Actual result

plugin.js:2921 Uncaught TypeError: Cannot read property '0' of null
    at Repository.fixCrossContentSelection (plugin.js:2921)
    at Repository.listenerFirer (event.js:144)
    at Repository.CKEDITOR.event.fire (event.js:290)
    at fireCheckSelection (plugin.js:2952)

## Other details

Fix: line 2921: check for null in /widget plugin.js
`range = ranges? ranges[ 0 ]:null;`

Fix line 292: check for null in /widget plugin.us
`if (!this.editor.getSelection()) return;`

* Browser: Chrome latest
* OS: Linux
* CKEditor version: 4.15.?
* Installed CKEditor plugins: widget

**Comment by jacekbogdanski** [comment]
It's the regression introduced in https://github.com/ckeditor/ckeditor4/commit/d633573567628308391da5a5f29e1a35e55b9b33 so between 4.13.0 and 4.13.1.

**Comment by f1ames** [comment]
Fixed in cb6f2c4.
